### DISCUSSION TITLE
Sponsorship of a fighter who is fighting live on espn

### DISCUSSION THREAD

**Discussion by 31trainman** [akash-network/community#467] [General]
Would the people of akash network be interested in sponsoring a fight that is being live streamed on espn? The potential amount of viewers of this fight presents a huge opportunity to bring new users to akash network.

The fighters are Chris metel and Omar Sandoval. The fight will be on march 16th.

**Comment by nick134-bit** [comment]
I personally have reservations about one-time commercial events, especially when it comes to crypto projects. However, I don't watch any ESPN events, and there might be a overlap between their viewership and potential Akash users that I may not be fully aware of

**Reply by 31trainman** [reply]
What sort of reservations do you have?

**Comment by rodri-r** [comment]
I think that if within budget this would be awesome. 

I spoke about this with @CaptainSplince  when we first met over two years ago now.  Akash signs and logo everywhere.

I see the boxer with black trunks and a red Akash logo,as well as a black boxing robe with the logo, walking down in between the public.  Would be awesome to see this boxer's corner team with some Akash shirts and merch too!

**Reply by rodri-r** [reply]
When we met online 😁

**Reply by rodri-r** [reply]
Would need to show the Akash Network url somewhere though, so people know what the logo is. 

Maybe a short commercial in between rounds or when they announce the boxer, something like "boxer's presentation and then sponsored by Akash Network - The Unstoppable Cloud - https://akash.network/ "

**Reply by rodri-r** [reply]
Would be really cool if the boxer's nickname was "Unstoppable his/her name" too and even better if he/she knocks out their rival in round 1

**Comment by hewi333** [comment]
Hi @31trainman thanks for bringing this idea up. Adding some additional detail might be helpful before considering something like this. I think you have a lot of great ideas but without details, its difficult to act upon it. My initial questions are:

1. How much would this cost?
2. What does it include (terms)?
3. Do you have a relationship with these fighters?
4. Are you wanting to submit a governance proposal?
5. Who is the point of contact? Are you setting up agreements?
6. Who would be signing a contract?
7. Who are we targeting? (I'm assuming retail investors not developers?)
8. What kind of exposure does this get us? (views/impressions/etc.)
9. Who will make the robe/shorts/marketing materials/graphics/etc?

**Comment by brewsterdrinkwater** [comment]
The community passed on this opportunity.
